spec/directions/bake_figure_spec.rb

```diff
@@ -42,6 +62,7 @@
               <span class="os-title-label">Figure </span>
               <span class="os-number">1.2</span>
               <span class="os-divider"> </span>
+              <span class="os-title" data-type="title" id="">This Is A Title</span>
```

Comment:
So it turns out that data-type="title" and figcaption represent _very **very** similar_ things, but not the _same_ thing. I fixed that and fixed figures & spec to match.

Reply:
Thank you for this comment.